src/main/java/com/microsoft/azure/vmagent/AzureVMManagementServiceDelegate.java

```diff
@@ -2232,7 +2232,7 @@ public String verifyStorageAccountName(
                 return Constants.OP_SUCCESS;
             }
         } catch (Exception e) {
-            LOGGER.log(Level.INFO, e.getMessage());
+            LOGGER.log(Level.SEVERE, "Fail to verify storage account name", e);
```

Comment:
with more information,  such as link to storage naming convention? `Fail to verify` seems something wrong with verifying itself. how about `Verification failed` or something better?

Reply:
No need to provide naming convention here since it is the log used for debug, users cannot see it normally. Will update the summary of the exception.

Reply:
what will user see in this case?

Reply:
This will occur only when something unexpected happens. And these error messages only exist in the jenkins log file, not in the Console Output of each Jenkins run. User's error message is like [this](https://github.com/jenkinsci/azure-vm-agents-plugin/blob/dab5150746fcd6f7c0e59fcdda1628ed5ed40d94/src/main/java/com/microsoft/azure/vmagent/AzureVMManagementServiceDelegate.java#L2237).